woocommerce/woocommerce-ios
# [Woo POS] Update products list UI

Closes: #13414


## Description


Updated main parts of the Products list view
- updated info banner view
- updated icons
- updates fonts and colors
- added shadow
- making content full width and height and make background colors go behind safe area

**Disclaimer**: some shadows, paddings and similar things might need tweaks later on when we update other parts of the POS UI

## Testing information


- Enter POS
- Make sure the products are shown
- Close the top info banner
- Tap few products and check that they are added to the cart

## Screenshots


| Before      | After |
| ----------- | ----------- |
|   ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 55 57](https://github.com/user-attachments/assets/1c2708ff-d18b-4595-9eb7-34ee4a27d510)    | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 50 55](https://github.com/user-attachments/assets/e3254fbf-7c6c-4ec8-944a-8c9baf57716f)       |
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 44 02](https://github.com/user-attachments/assets/f50964b9-f16f-4d57-910d-f3171502f090)      | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 50 58](https://github.com/user-attachments/assets/e6cc966b-dec8-41c0-95e3-5b3ec45f04ac)       |
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 44 10](https://github.com/user-attachments/assets/b750d528-e627-48d5-aa95-8fd66d8f9705)     | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-07-25 at 10 51 03](https://github.com/user-attachments/assets/233cf742-1cea-4cad-a6ac-f095fc8e7edd)       |

---
